frameworks/eyra_ui/assets/css/app.css

```diff
@@ -0,0 +1,113 @@
+@tailwind base;
```

Comment:
Do we need to keep this app.css in sync with the one in Core?

Reply:
Yes, this is a side-effect of having the Eyra UI code in a seperate folder. We could have a look into making the packager smart enough to handle it. Another option would be to move Eyra UI into core (`lib/eyra_ui`). Do you have a preference?